apps/minifront/src/components/staking/account/delegation-value-view/staking-actions/index.tsx

```diff
@@ -41,17 +41,11 @@ export const StakingActions = ({
   const state = useStoreShallow(stakingActionsSelector);
   const validator = getValidator(validatorInfo);
 
-  const stakingTokensAndFilter = useStakingTokensAndFilter();
-  const unstakedTokensForThisAccount = stakingTokensAndFilter.data?.unstakedTokensByAccount.get(
-    state.account,
-  );
+  const { stakingTokens } = useStakingTokensAndFilter(state.account);
```

Comment:
See my note in the test file above — `stakingTokens` already represents the staking tokens for this account, rather than a map.